dusty/systems/virtualbox/__init__.py

```diff
@@ -16,7 +18,12 @@
 def _run_command_on_vm(command_list):
     ssh_command = ['docker-machine', 'ssh', constants.VM_MACHINE_NAME]
     ssh_command.extend([command_list])
-    check_and_log_output_and_error_demoted(ssh_command)
+    result = check_and_log_output_and_error_demoted(ssh_command)
+    return result
+
+def vm_current_time():
```

Comment:
If we're only using this for tests, this function should go in a test specific location IMO